invenio_rdm_records/resources/serializers/dublincore/__init__.py

```diff
@@ -34,12 +34,18 @@ class DublinCoreXMLSerializer(MarshmallowSerializer):
     records.
     """
 
-    def __init__(self, **options):
+    def __init__(self, schema_context=None, **options):
         """Constructor."""
+        # note: the superclass init will store the passed `schema_context` for
+        # later use (this controls some behavior of the `DublinCoreSchema` when
+        # dumping) and pass the remaining `options` to the `SimpleSerializer`.
+        # because the latter only accepts an `encoder` argument, we need to
+        # ignore the remaining keyword arguments.
+
         super().__init__(
             format_serializer_cls=SimpleSerializer,
             object_schema_cls=DublinCoreSchema,
             list_schema_cls=BaseListSchema,
+            schema_context=schema_context or {},
             encoder=simpledc.tostring,
-            **options,
```

Comment:
minor/question: aren't we now missing the ability to pass additional `options` to the `format_serializer_cls`? Well, as you wrote, for `SimpleSerializer` there's not much more actually to pass, but if that changes in the future, we'll have to add it back, and I think `**options` gets merged with `encoder` in the `**serializer_options` of the superclass (so theoretically no harm done).

Reply:
Yes, the configuration flexibility will suffer a little bit from this change...

Unfortunately, giving unexpected arguments via `**serializer_options` will cause a `TypeError` ([c.f. the app-rm test failure](https://github.com/inveniosoftware/invenio-app-rdm/actions/runs/4545299972/jobs/8012434399#step:9:217)) in the `format_serializer_cls` constructor, rather than ignoring the unexpected args:

```python
>       self.format_serializer_cls = format_serializer_cls(**serializer_options)
E       TypeError: __init__() got an unexpected keyword argument 'options'
```
with:
```python
serializer_options = {'encoder': <function tostring at 0x7f3befb3cc20>, 'options': {'indent': 2, 'sort_keys': True}}
```

As an alternative to this PR here, we could also teach the `SimpleSerializer` to take and ignore extra keyword args :smile:

Reply:
Well, would you look at that: https://github.com/inveniosoftware/flask-resources/pull/116/files
When that's merged in, we can actually close this one here